apps/web-giddh/src/app/material-ledger/components/share-ledger/share-ledger.component.scss

```diff
@@ -0,0 +1,12 @@
+@import "../../../../assets/css/color.scss";
+
+.btn-success:disabled {
+    color: $white !important;
+}
+.mat-form-field {
+    width: 80% !important;
+    margin-top: 2px;
+}
+.pd-0 {
+    padding-left: 5px !important;
```

Comment:
This is not as per convention as it is for `padding: 0`. Consider renaming the class for padding.

Reply:
We created a new class pd-0 in style-2.scss for padding: 0 and we are overriding it here.